README.md

```diff
@@ -49,21 +49,18 @@ An example implementation of can be found in [middleware/validateHeapHeader.ts](
 
 This will be a list of users to add into a certain segment. See [bin/data/sync.json](./bin/data/sync.json) for an example payload.
 
-### Users Drain
-
-This will be a segment ID that should be drained (for cases where the data has gotten out of sync).  See [bin/data/drain.json](./bin/data/drain.json) for an example payload.
```

Comment:
Should we remove drain.json as well?

Reply:
oops i didnt see this, i was just gonna leave it with the _slim_ change we work on it in the future